semeio/jobs/scripts/replace_string.py

```diff
@@ -13,3 +18,34 @@
 """
 
 category = "utility.templating"
+
+
+def _get_args_parser():
+
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument(
+        "-o",
+        "--original",
+        help="Original text",
+    )
+    parser.add_argument(
+        "-n",
+        "--new",
+        help="New text",
+    )
+    parser.add_argument(
+        "-f",
+        "--file",
+        help="File to where replacement should occur",
+    )
+    return parser
+
+
+def main_entry_point():
+    parser = _get_args_parser()
+    options = parser.parse_args()
+    file = Path(options.file)
+    file.write_text(
+        file.read_text().replace(options.original.strip("'"), options.new.strip("'"))
```

Comment:
Is it necessary to strip away `'`? Very common character in Eclipse files, which I would think it is likely is one of the main use cases for this job.

Reply:
I guess it is not necessary, removed the stripping :)